recipe/meta.yaml

```
@@ -1,42 +1,35 @@
-{% set build = 0 %}
 {% set name = "libheif" %}
 {% set version = "1.19.5" %}
-# ensure license_family is defined to appease bot
-{% set license_family = license_family or 'notset' %}
-
-{%- if license_family == 'gpl' %}
-{%- set build = build + 100 %}
-{%- endif %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/strukturag/libheif/releases/download/v{{ version }}/libheif-{{ version }}.tar.gz
+  url: https://github.com/strukturag/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
   sha256: d3cf0a76076115a070f9bc87cf5259b333a1f05806500045338798486d0afbaf
 
 build:
-  number: {{ build }}
-  string: {{ license_family }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  # x265 not available for s390x
+  skip: True  # [s390x]
+  number: 0
   run_exports:
-    - {{ pin_subpackage('libheif', max_pin='x.x') }}
+    - {{ pin_subpackage('libheif', max_pin='x.x.x') }}
 
 requirements:
   build:
     - gnuconfig  # [unix]
     - {{ compiler('cxx') }}
-    - {{ stdlib('c') }}
     - cmake
     - ninja
   host:
     - libde265
-    - libavif
-    # - svt-av1
-    # - rav1e
     - dav1d
     - aom
```

```suggestion
    - dav1d 1.2.1
    - aom  3.6
```
Pin the host dependencies

---

Done